Makefile

```
@@ -445,6 +445,30 @@ community-operators-bundle: bundle ## Prepare bundle for community-operators-pro
 	fi
 	@echo "Bundle prepared at: community-operators-prod/operators/automotive-dev-operator/$(VERSION)"
 
+.PHONY: bump-go
+bump-go: ## Update Go version everywhere (usage: make bump-go GO_PATCH_VERSION=1.26.3)
```

checkmake (0.3.2)

[warning] Target body for "bump-go" exceeds allowed length of 5 lines (21).

(maxbodylength)